Integrations/integration-ThreatMiner.yml

```diff
@@ -0,0 +1,863 @@
+  md += '\n'
+  md += '**File SHA256:** {}'.format(FileContext['ThreatMiner']['File'].get('Sha256','Unkown'))
+  md += '\n'
+  md += '**File Type:** {}'.format(FileContext['ThreatMiner']['File'].get('Type','Unkown'))
+  md += '\n'
+  md += '**File Name:** {}'.format(FileContext['ThreatMiner']['File'].get('FileName','Unkown'))
+  md += '\n'
+  md += '**File Size:** {}'.format(FileContext['ThreatMiner']['File'].get('Size','Unkown'))
+  md += '\n'
+  md += '**File Analyzed:** {}'.format(FileContext['ThreatMiner']['File'].get('Analyzed','Unkown'))
+
+
+  if len(FileContext['ThreatMiner']['File'].get('HTTP','')) != 0:
+    md += tableToMarkdown("HTTP for hashed file {}".format(hashedFile), FileContext['ThreatMiner']['File']['HTTP'], ['Domain', 'URL', 'UserAgent'])
+    threadMinerFoundResults = True
+  if len(FileContext['ThreatMiner']['File'].get('Domains','')) != 0:
+    md += tableToMarkdown("Hashed file: {} Domains".format(hashedFile),  FileContext['ThreatMiner']['File']['Domains'], ['domain','ip'])
+    threadMinerFoundResults = True
+  if len(FileContext['ThreatMiner']['File'].get('Mutants','')) != 0:
+    md += tableToMarkdown("Hashed file: {} Mutants".format(hashedFile),  FileContext['ThreatMiner']['File']['Mutants'], ['Mutants'])
+    threadMinerFoundResults = True
+  if len(FileContext['ThreatMiner']['File'].get('Registry','')) != 0:
+    md += tableToMarkdown("Hashed file: {} Registry keys".format(hashedFile), FileContext['ThreatMiner']['File']['Registry'], ['Registry'])
+    threadMinerFoundResults = True
+  if len(FileContext['ThreatMiner']['File'].get('AV','')) != 0:
+    md += tableToMarkdown("Hashed file: {} Anti Virus detections".format(hashedFile), FileContext['ThreatMiner']['File']['AV'], ['av','detection'])
+    threadMinerFoundResults = True
+
+  if threadMinerFoundResults == False:
+    md += 'No results found'
+
+  return md
+
+def delete_proxy_if_asked():
+  if not demisto.params()['proxy']:
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+  ''' EXECUTION CODE '''
+try:
+  demistoCommand = demisto.command()
+  if demistoCommand == 'test-module':
+    report = get_ip_whois_rawdata('8.8.8.8')
+
+    if 'asn' in report:
+      demisto.results('ok')
+    else:
+      demisto.results('test failed')
+    pass
+
+  if demisto.params().get('limitResults').lower() == 'all':
+    maxArraySize = -1
+  else:
+    maxArraySize = int(demisto.params().get('limitResults', MAX_RETURNED_ARRAY_SIZE))
+
+  delete_proxy_if_asked()
+
+  if demistoCommand == 'domain':
+    domain_command(maxArraySize)
+
+  if demistoCommand == 'ip':
+    ip_command(maxArraySize)
+
+  if demistoCommand == 'file':
+    file_command(maxArraySize)
+
+except Exception, e:
+  raise
+
+  type: python
+  commands:
+  - name: domain
+    arguments:
+    - name: domain
+      required: true
+      description: Domain name
+    outputs:
+    - contextPath: ThreatMiner.Domain
```

Comment:
You are missing the standard context outputs
```
Domain.Address
Domain.WHOIS.Server
....
```
please take a look here https://github.com/demisto/content/blob/master/docs/context_standards/README.MD